src/main/scala/geekie/mapred/io/OutputWriterFileClosed.scala

```diff
@@ -0,0 +1,4 @@
+package geekie.mapred.io
+
+
+case object OutputWriterFileClosed
```

Comment:
I think the usual is to put these specific messages inside the companion object of the corresponding actor class. Then we can leave everything inside the `OutputWriter.scala` file. The problem is you have to import the message object in the other classes.